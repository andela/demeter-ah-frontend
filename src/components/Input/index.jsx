import React from 'react';
import './index.scss';

const Input = ({ inputType, classes, ...rest }) => (
  <input type={inputType} required className={`forminput rounded py-2 bg-purple-150 ${classes}`} {...rest} />
);

export default Input;
