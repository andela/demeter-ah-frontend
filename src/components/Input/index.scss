.forminput{
  &::placeholder {
    color: rgb(92, 91, 91);
    font-size: 0.9em;
  }
}