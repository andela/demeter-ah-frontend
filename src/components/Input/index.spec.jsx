import React from 'react';
import { shallow } from 'enzyme';
import Input from './index';

describe('Input Component', () => {
  it('Should render without errors', () => {
    const component = shallow(<Input />);
    expect(component).toMatchSnapshot();
  });
});
