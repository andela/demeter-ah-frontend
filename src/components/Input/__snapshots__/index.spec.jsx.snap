// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component Should render without errors 1`] = `
<input
  className="forminput rounded py-2 bg-purple-150 undefined"
  required={true}
/>
`;
